lib/src/migrators/module.dart

```diff
@@ -434,27 +479,32 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
           "Migration of @import rule with multiple imports not supported.");
     }
     var import = node.imports.first as DynamicImport;
-    var migrateToLoadCss = _scope.parent != null || !_useAllowed;
 
     var oldConfiguredVariables = _configuredVariables;
-    _configuredVariables = Set();
-    _upstreamStylesheets.add(_currentUrl);
-
-    var oldScope = _scope;
-    if (migrateToLoadCss) {
-      _scope = oldScope.copyForNestedImport();
-      var current = oldScope.parent;
-      while (current != null) {
-        _scope.variables.addAll(current.variables);
-        current = current.parent;
+    _configuredVariables = {};
+    _upstreamStylesheets.add(currentUrl);
+    if (!_useAllowed) {
+      _unreferencable = UnreferencableMembers(_unreferencable);
+      for (var declaration in references.variables.values
+          .followedBy(references.functions.values)
+          .followedBy(references.mixins.values)) {
```

Comment:
Would it make sense to add `references.allDeclarations` as a convenience getter for `references.variables.values.followedBy(...).followedBy(...)`?